tests/roles/prelude_controller/tasks/main.yaml

```diff
@@ -0,0 +1,2 @@
+- name: check connection
+  ansible.builtin.ping:
```

Comment:
https://github.com/openstack-k8s-operators/data-plane-adoption/blob/main/docs/mariadb_copy.md#procedure---stopping-control-plane-services

See ^^. So creating the prelude_controller role was a bit premature, it may not be needed at all.

Reply:
(This area could be a candidate for actual end user automation later, but for now i'd like to stick to the docs procedure as closely as possible.)